scripts/script-blog.js

```diff
@@ -0,0 +1,70 @@
+/* Add Header to Page */
+document.getElementById("blog-h1").innerHTML = "Adopt A Dog";
+
+
+
```

Comment:
So one thing you could try to do is create a js object or array of objects for each article, then you could use a loop to generate the same html content for each object and add it to the page. As far as adding things without a blank html element to target, the procedure is largely the same. Instead of 
`document.getElementById('blog-1');`
You can instead target your body element, and 
`.appendChild()` for each item. if you need unique id's for them you can use the iterator of the loop to create a unique string. If you need more explanation we can explain this better at office hours.